src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ExtensionHttpFuzzer.java

```diff
@@ -153,6 +149,9 @@ public void hook(ExtensionHook extensionHook) {
                         extensionAntiCSRF));
             }
         }
+
+        httpFuzzerHandler.addFuzzerMessageProcessorUIHandler(new HttpFuzzerMessageProcessorTagUIHandler());
+        addFuzzResultStateHighlighter(new HttpFuzzerMessageProcessorTagStateHighlighter());
```

Comment:
The two statements should be moved to the previous if-then block, the processor and highlighter should be added only if there's GUI.